Neo4j.Driver/Neo4j.Driver/Internal/Routing/RoundRobinRoutingTable.cs

```diff
@@ -32,22 +32,23 @@ internal class RoundRobinRoutingTable : IRoutingTable
         private readonly Stopwatch _stopwatch;
         private readonly long _expireAfterSeconds;
 
-        public RoundRobinRoutingTable(Stopwatch stopwatch, long expireAfterSeconds = 0)
+        public RoundRobinRoutingTable(IEnumerable<Uri> routers, long expireAfterSeconds = 0)
```

Comment:
This constructor may call the other one using Enumerable.Empty<Uri>() for readers and writers parameters.